modules/dcache/src/main/java/diskCacheV111/services/TransferManager.java

```diff
@@ -475,7 +397,8 @@ public void setBilling(CellStub billingStub) {
     }
 
     @Autowired(required = false)
-    public void setTransferTemplate(KafkaTemplate kafkaTemplate) {
+    @Qualifier("")
```

Comment:
There are different beans of type KafkaTemplate and it was complaining about not knowing which one to use. 
I've removed the qualifier annotation since I didn't specify one and builds successfully.